graph-oauth/src/accesstoken.rs

```diff
@@ -514,3 +516,53 @@ impl fmt::Debug for AccessToken {
             .finish()
     }
 }
+
+#[allow(unused_imports, dead_code)]
+#[cfg(test)]
+mod access_token_tests {
+
+    use super::*;
+    pub const ACCESS_TOKEN_INT  : &'static str = r#"{
+        "access_token": "fasdfasdfasfdasdfasfsdf",
+        "token_type": "Bearer",
+        "expires_in": 65874,
+        "scope": null,
+        "refresh_token": null,
+        "user_id": "santa@north.pole.com",
+        "id_token": "789aasdf-asdf",
+        "state": null,
+        "timestamp": "2020-10-27T16:31:38.788098400Z"
+}"#;
+pub const ACCESS_TOKEN_STRING  : &'static str = r#"{
+    "access_token": "fasdfasdfasfdasdfasfsdf",
+    "token_type": "Bearer",
+    "expires_in": "65874",
+    "scope": null,
+    "refresh_token": null,
+    "user_id": "helpers@north.pole.com",
+    "id_token": "789aasdf-asdf",
+    "state": null,
+    "timestamp": "2020-10-27T16:31:38.788098400Z"
+}"#;
+
+
+    #[test]
+    pub fn test_deserialize() {
+        let _token: AccessToken = serde_json::from_str(ACCESS_TOKEN_INT).unwrap();
+        let _token: AccessToken = serde_json::from_str(ACCESS_TOKEN_STRING).unwrap();
+    }
+
+    #[test]
+    pub fn test_serialize_token() {
```

Comment:
We defineteley want to make sure both scenarios are covered but there are plenty of tests that already cover the expires_in field with it being an int. Can you remove this test and then your other test should be moved to https://github.com/sreeise/graph-rs/blob/master/tests/access_token_tests.rs

Reply:
Done